src/main/java/org/truffleruby/core/cast/ToStrNode.java

```diff
@@ -43,7 +44,7 @@ public DynamicObject coerceObject(VirtualFrame frame, Object object,
             @Cached("create()") BranchProfile errorProfile) {
         final Object coerced;
         try {
-            coerced = toStrNode.call(frame, object, "to_str");
+            coerced = getToStrNode().call(frame, object, "to_str");
```

Comment:
Can it be created in `@Cached` or is it better like this for footprint?

Reply:
fixed